notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb

```diff
@@ -0,0 +1,1230 @@
+{
```

Comment:
<p>combine the 2 code cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
done